build.gradle

```diff
@@ -55,13 +51,10 @@ allprojects {
     }
 
     repositories {
-        mavenCentral()
-        maven {
-            url 'https://dl.bintray.com/palantir/releases/'
-        }
-        maven {
-            url 'https://dl.bintray.com/marshallpierce/maven/'
-        }
+        jcenter()
+        gradlePluginPortal()
```

Comment:
I don't think we want the gradle plugin portal resolver available for project dependency resolution, do we?

Reply:
good call, was copy paste from https://github.com/palantir/gradle-baseline/blob/develop/build.gradle#L32 (which makes sense since that is building gradle plugins)